src/molecules/side-menu.module.css

```
@@ -0,0 +1,92 @@
+.sideMenu {
+    position: fixed;
+    z-index: 10;
+    top: 0;
+    left: 0;
+    width: 22.5rem;
+    height: 100%;
+    transform: translateX(-100%);
+    transition: transform 0.15s ease-in-out;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    display: flex;
+    flex-direction: column;
+    border-top-right-radius: 1rem;
+    border-bottom-right-radius: 1rem;
+  }
+  
+.sideMenu[data-open="true"] {
+    transform: translateX(0);
+}
+.sideMenu[data-open="false"] {
+  transform: translateX(-100%);
+}
+  .overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    opacity: 0.2;
+    z-index: 5;
+  }
+  
+  .topBar {
+    padding: 1.5rem;
+    display: flex;
+  align-items: end;
+  justify-content: space-between;
+  }
+  
+  .closeButton {
+    font-size: 1.5rem;
+    color: var(--on-surface-color);
+    background: none;
+    border: none;
+    cursor: pointer;
+    transform: translateY(11.5%);
+  }
+  
+.menuItemButton {
+    padding: 1rem;
+    margin: 0.5rem;
+    border: none;
+    border-radius: 9999px;
+    cursor: pointer;
+    text-align: left;
+}
+  
+.menuItemButton:hover {
+    background-color: color-mix(in srgb, currentcolor 8%, transparent);
+}
+  
+  .subjectList {
+    overflow-y: auto;
+    flex-grow: 1;
+    display: flex;
+    flex-direction: column;
+  }
+  
+  .subjectItem {
+    padding: 1rem;
+    margin: 0.5rem 0;
+    background-color: #FCE4EC;
+    color: #000;
+    border-radius: 0.5rem;
+    display: flex;
+    justify-content: space-between;
+  } 
+  
+  .subjectItem.highlight {
+    background-color: #fde4ef;
+  }
+
+  .subjectName {
+    font-weight: bold;
+    color: #333;
+    margin-bottom: 2px;
+  }
+  
+  .lastDate {
+    color: #999;
```

```suggestion
    border-radius: 0.5rem;
    display: flex;
    justify-content: space-between;
  } 
  
  .subjectItem.highlight {
  }

  .subjectName {
    font-weight: bold;
    margin-bottom: 2px;
  }
  
  .lastDate {
```
コンポーネント側でクラスの割り当てを書いていますから, こちらの色指定は除去してください